erc20-connector/contracts/BridgeTokenFactory.sol

```diff
@@ -79,6 +79,8 @@ contract BridgeTokenFactory is ERC20Locker {
 
     function _decodeLockResult(bytes memory data) internal pure returns(LockResult memory result) {
         Borsh.Data memory borshData = Borsh.from(data);
+        uint8 memory flag = borshData.decodeU8();
```

Comment:
Remove `memory`, `uint8` is a value type so it doesn't require storage specifier.

Reply:
🆗 

I still don't understand how is memory managed in solidity, I used `memory` from my understanding after reading here:
https://solidity.readthedocs.io/en/develop/introduction-to-smart-contracts.html#storage-memory-and-the-stack

So from you comment I understand that it is similar to rust traits `Copy` and `Clone`, in this case some basic types (such as `uint8`) is `Copy` so its value is not stored in `storage` but instead is directly stored in `memory`, without needing to specify it.